whitelabelco/sofo-foods
# Cp/feat/change background tan

### Description 💻

Resolves: #153 

### Implementation 💡

As requested by client, replace tan background color with `#f0f0f0` grey
There are three instances where this color is used. See testing steps


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- see Hello from Sofo section
- scroll down to Customer Service callout
- click "About Us"
- see youtube video background

### Screenshots/Demos 📷

### Checklist ✅
